i18n/data.json

```diff
@@ -24,7 +24,39 @@
 		"avatar.upload": "Add image",
 		"avatar.confirm": "Set picture",
         "theme.light": "Light",
-        "theme.dark": "Dark"
+        "theme.dark": "Dark",
+        "server.userConnected": "{nickname} connected",
+        "server.userDisconnected": "{nickname} disconnected",
+        "server.newRule": "New rule: ",
+        "rule.anonymity.title": "Anonymity",
+        "rule.anonymity.description": "Hides the identity of players.",
+        "rule.messageMinLength.title": "Message Minimum Length Limit",
+        "rule.messageMaxLength.title": "Message Maximum Length Limit",
+        "rule.messageMinLength.description": "Restricts message minimum length.",
+        "rule.messageMaxLength.description": "Restricts message maximum length.",
+        "rule.posMinLimit.title": "Minimum Number of Verbs/Nouns/Adjectives",
+        "rule.posMinLimit.description": "Sets the minumum number of verbs/nouns/adjectives in message.",
+        "rule.posMaxLimit.title": "Maximum Number of Verbs/Nouns/Adjectives",
+        "rule.posMaxLimit.description": "Sets the maximum number of verbs/nouns/adjectives in message.",
+        "rule.haiku.title": "Haiku Messages",
+        "rule.haiku.description": "All messages must have 3 verses (separated by / or new line) with 5, 7, and 5 syllables.",
+        "rule.noMetre.title": "No Metre",
+        "rule.mute.title": "Mute Player",
+        "rule.mute.description": "Mutes a specific player.",
+        "rule.chatTurns.title": "Chat Turns",
+        "rule.chatTurns.description": "Players can only speak during their turn.",
+        "rule.noChatTurns.title": "Disable Chat Turns",
+        "rule.noMessageLength.title": "Disable message length",
+        "rule.unmute.title": "Unmute",
+        "rule.disableFormatting.title": "Disable MarkDown formatting",
+        "rule.returnNames.title": "Return original names",
+        "rule.noPosLimit.title": "Disable POS limit",
```

Comment:
This could probably be a bit more descriptive. POS doesn't really ring a bell. Maybe something like
```
Disable parts of sentence limit
```
or
```
Disable verbs/nouns/adjectives limit
```
?

Reply:
Those translations were just copy pasted from the server's translation files, so this is kind of outside of the scope of the PR, but I fixed it anyway